styleguide/components/41-organisms/document-box/document-box.twig

```diff
@@ -0,0 +1,49 @@
+<div class="document-box accordion">
+  {% include '@h2' with {
+    'h2_text': 'Documents'
+  } %}
+  <ul class="links">
+    {% for document in documents %}
+      <li>
+        {% block link %}
+          {% include '@link' with {
+            'link': 'javascript:(void);',
+            'text': document.text,
+            'download': true,
+            'modifier': 'standalone-link'
+          } %}
+          <span class="file-type">
+            {{ document.type }}<span class="file-size">({{ document.size }})</span>
+          </span>
+        {% endblock %}
+      </li>
+    {% endfor %}
+  </ul>
+
+  <h3>
+    <button class="accordion--button"
+            aria-expanded="false"
+            aria-controls="documents--content">
+      {{ 'There are 5 other documents available' }}
```

Comment:
Better not hard-code this, count the document array.

Reply:
Fixed.